src/cap-description.adoc

```diff
@@ -188,6 +188,11 @@ reserved values as if it were 0b00000 (no permissions). Future extensions may as
 meanings to the reserved bit patterns, in which case <<GCPERM>> is allowed to report a
 non-zero value.
 
+NOTE: When MXLEN=32 the hart's CHERI execution mode is always Capability (see
+xref:m_bit[xrefstyle=short]), so the encodings for Legacy Mode are reserved,
+and must not appear in valid capabilities, if only {cheri_base_ext_name} is
+supported.
+
```

Comment:
why is this here? It should start with "if only Zcheri_purecap is supported" to make the intent clearer, but do we need to say this? Surely this text is elsewhere, when we instructroduce Zcheri_purecap / Zcheri_legacy?

Reply:
I could not find this kind of text elsewhere. The closest is perhaps this from the Zcheri_legacy chapter:
> NOTE: The implication is that the CHERI execution mode is always Capability on
implementations that support {cheri_base_ext_name}, but not
{cheri_legacy_ext_name}.

I think the main point here is to indicate that the AP encodings for Legacy are "reserved" when only Zcheri_purecap is supported.

Happy to make the text clearer once we agree whether its needed and where it should go.